src/collections/_documentation/api/projects/put-project-details.md

```diff
@@ -1,12 +1,12 @@
+  "example_response": "HTTP/1.1 500 Internal Server Error\nContent-Length: 72\nX-XSS-Protection: 1; mode=block\nX-Content-Type-Options: nosniff\nContent-Language: en\nAccess-Control-Expose-Headers: X-Sentry-Error, Retry-After\nVary: Accept-Language, Cookie\nAccess-Control-Allow-Methods: GET, PUT, DELETE, HEAD, OPTIONS\nAllow: GET, PUT, DELETE, HEAD, OPTIONS\nAccess-Control-Allow-Origin: *\nAccess-Control-Allow-Headers: X-Sentry-Auth, X-Requested-With, Origin, Accept, Content-Type, Authentication, Authorization\nContent-Type: application/json\nX-Frame-Options: deny\n\n{\n  \"detail\": \"Internal Error\", \n  \"errorId\": \"fa56e725d03841228d820112fcada54b\"\n}", 
```

Comment:
I spy with my little eye - a 500.

Reply:
This doesn't happen if you just run `python api-docs/generator.py --output-path=/tmp/src/output --output-format=both` in sentry. Probably something's up with our docker setup.

Reply:
Okay, so when the associated scenario is run, we get this error message:
```
Traceback (most recent call last):
  File "/usr/src/sentry-meredith-test-fixing-apidoc-generator/src/sentry/api/base.py", line 89, in handle_exception
    response = super(Endpoint, self).handle_exception(exc)
  File "/usr/local/lib/python2.7/site-packages/rest_framework/views.py", line 449, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/src/sentry-meredith-test-fixing-apidoc-generator/src/sentry/api/base.py", line 196, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/src/sentry-meredith-test-fixing-apidoc-generator/src/sentry/api/endpoints/project_details.py", line 371, in put
    if not serializer.is_valid():
  File "/usr/local/lib/python2.7/site-packages/rest_framework/serializers.py", line 237, in is_valid
    self._validated_data = self.run_validation(self.initial_data)
  File "/usr/local/lib/python2.7/site-packages/rest_framework/serializers.py", line 432, in run_validation
    value = self.to_internal_value(data)
  File "/usr/local/lib/python2.7/site-packages/rest_framework/serializers.py", line 464, in to_internal_value
    validated_value = validate_method(validated_value)
  File "/usr/src/sentry-meredith-test-fixing-apidoc-generator/src/sentry/api/endpoints/project_details.py", line 282, in validate_platform
    if Project.is_valid_platform(value):
  File "/usr/src/sentry-meredith-test-fixing-apidoc-generator/src/sentry/models/project.py", line 432, in is_valid_platform
    return integration_doc_exists(value)
  File "/usr/src/sentry-meredith-test-fixing-apidoc-generator/src/sentry/utils/integrationdocs.py", line 146, in integration_doc_exists
    docs = os.listdir(DOC_FOLDER)
OSError: [Errno 2] No such file or directory: '/usr/src/sentry-meredith-test-fixing-apidoc-generator/src/sentry/integration-docs'
```

This happens because we install a light build (`SENTRY_LIGHT_BUILD=1`), in which `build_integration_docs` [does not run](https://github.com/getsentry/sentry/blob/master/setup.py#L77). So, there is no `DOC_FOLDER`. The `is_valid_platform` bit is because of https://github.com/getsentry/sentry/pull/16079.

Reply:
Could the integration docs be built right after sentry is installed with `pip install -e '.[dev]'`?

Reply:
Spoke to @markstory on DMs but putting it here for visibility.

I tried running `make build-platform-assets` after the pip step and get this error:
```
root@bb0e62d2a17b:/usr/src/sentry-meredith-test-fixing-apidoc-generator# make build-platform-assets
--> Building platform assets
Error: Configuration file does not exist. Use 'sentry init' to initialize the file.
Makefile:112: recipe for target 'build-platform-assets' failed
make: *** [build-platform-assets] Error 1
```

Since we need `sentry init` to run [`sync_docs`](https://github.com/getsentry/sentry/blob/master/Makefile#L116), probably throws option 2, 3 & 4 out the window. Going to go with option 1 for now.